Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -161,14 +169,13 @@ inline void ClearPath() {
     }
 };
 
-#define NOISY_RETURN_IF_NULL(param, ...)                                         \
-    do {                                                                         \
-        if (!context) {                                                          \
+#define NOISY_RETURN_IF_NULL(param, ...)                                    \
+    do {                                                                    \
+        if (!context) {                                                     \
```

Comment:
move this a generic file, so all of us call use it :)

result.h might be a good destination. 

---
In reply to: [85814159](https://github.com/Microsoft/WinObjC/pull/1239#discussion_r85814159) [](ancestors = 85814159)